galera/src/replicator_str.cpp

```diff
@@ -455,100 +455,86 @@ void ReplicatorSMM::process_state_req(void*       recv_ctx,
     wsrep_seqno_t rcode (0);
     bool join_now = true;
 
-    if (streq->ist_len())
+    if (!skip_sst)
```

Comment:
Note that this additional condition, and the shortening of the inner (original) condition is the only actual change in this PR, and this is how it looked like in galera 4.3. Everything else is a reformatting because of the additional nesting.